core/controllers/library.py

```diff
@@ -57,7 +57,7 @@ def get_matching_activity_dicts(query_string, search_cursor):
             collection_ids))
     activity_list += (
         summary_services.get_displayable_exp_summary_dicts_matching_ids(
-            exp_ids))
+            exp_ids, user_services.UserActionsInfo()))
```

Comment:
Here and below: I don't think this makes sense. Better to pass None.

Also, if you prefer, I think that with the new UserActions object it's OK for the called function to have a second arg that defaults to None, if necessary. The objection I had to default args before was that there were multiple args that could independently be set to default and that was conceptually muddy.